watchfaces/002-analog-70s-classic.qml

```diff
@@ -58,7 +58,10 @@ Item {
             top: dayDisplay.bottom
             topMargin: parent.height * 0.0156
         }
-        text: wallClock.time.toLocaleString(Qt.locale(), "<b>HH</b>:mm")
+        text: if (use12H.value) {
+                  wallClock.time.toLocaleString(Qt.locale(), "hh ap").slice(0, 2) + wallClock.time.toLocaleString(Qt.locale(), ":mm") }
+              else
+                  wallClock.time.toLocaleString(Qt.locale(), "HH:mm")
```

Comment:
it looks like you dropped the <b> </b> ?

Reply:
If you mean i dropped the bold tags. Yes, those where having no use since the League spartan font can not be made heavier that way. It could be manipulated with the font.weight: value but as it is now, there should be no change visually.

Reply:
ah alright :) if there is no visual change then it's all good